traffic_ops/testing/api/v3/profiles_test.go

```diff
@@ -90,8 +89,7 @@ func GetTestProfilesIMS(t *testing.T) {
 			if len(respParameter) > 0 {
 				parameterID := respParameter[0].ID
 				if parameterID > 0 {
-					t.Errorf("Parameter ID %d", parameterID)
-					resp, _, err := TOSession.GetProfileByParameterWithHdr(strconv.Itoa(parameterID), header)
+					resp, _, err := TOSession.GetProfileByParameterWithHdr(parameterID, nil)
```

Comment:
You can't actually change the existing client signatures because that breaks the clients that are using them. I suggest creating a new method that takes in the parameterID, instead of the parameter name, and then using that method in the tests. You can leave the `GetProfileByParameterWithHdr` method as it is.

Reply:
reverted the method signature